recipes/minisat/all/conandata.yml

```diff
@@ -0,0 +1,9 @@
+sources:
+  "2.2.0":
+    url: "https://github.com/stp/minisat/archive/3db58943b6ffe855d3b8c9a959300d9a148ab554.zip"
+    sha256: "b17090cf4fdec179b545bdc371ba0760ad85cacfb855f0a89d827d87fd85ce63"
```

Comment:
This url should be https://github.com/niklasso/minisat/archive/refs/tags/releases/2.2.0.tar.gz

Reply:
I used this fork as it is still 2.2.0 but adds CMake support

Reply:
Given that this CMakeLists is not very good and that in few lines another better one can be written, I would advise to download from official repo and export a custom CMakeLists.